tensorflow/core/common_runtime/eager/attr_builder_test.cc

```diff
@@ -165,15 +165,15 @@ TEST(AttrBuilder, BuildNodeDef_Modified) {
   a.NumInputs(2);
 
   const NodeDef& node_def = a.BuildNodeDef();
-  EXPECT_EQ(node_def.attr().size(), 2);
+  EXPECT_EQ(node_def.attr().size(), 4);
```

Comment:
Can you add the other attr's to this test?

Reply:
Done!